resolver/contentResolver.go

```diff
@@ -3,9 +3,10 @@ package resolver
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/Financial-Times/transactionid-utils-go"
 	"io/ioutil"
 	"net/http"
+
+	"github.com/Financial-Times/transactionid-utils-go"
 )
 
 type ContentResolver interface {
```

Comment:
Test the flow where the httpClient can't make the call to the contentResolverApp.

Reply:
Cannot test this, because of go not having an interface for http client.